docs/user-guide/scalability.md

```diff
@@ -0,0 +1,22 @@
+# Scalability
+
+Our findings and recommendations around scaling the starter kit are based on a set of load tests that we ran in January 2022. If you want to learn about how to run these load tests, or read up on a more detailed analysis, refer to the [Load Tests](./testing/load_tests.md) documentation. During all of the scenarios below we verified that the starter kit operated reliably, did not miss more than a few receive windows and did not drop any upstream messages.
```

Comment:
Let's specify that the tests were run on release 2.0.0 with a specific IoT Edge Hub version (don't recall exactly now)